lib/util.ml

```diff
@@ -1721,19 +1721,29 @@ let child_of_parent conf base p =
     [Rem]    : Exporté en clair hors de ce module.                           *)
 (* ************************************************************************** *)
 let husband_wife conf base p all =
-  let relation =
-    let rec loop i =
+  let multiple =
+    let rec loop i kind =
       if i < Array.length (get_family p) then
         let fam = foi base (get_family p).(i) in
+        let cur_type = get_relation fam in
+        if i = 0 then loop (i + 1) cur_type
+        else if cur_type = kind then loop (i + 1) kind
+        else -1
```

Comment:
I don't understand this line, are we assuming all relations are of the same kind?